OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -165,18 +167,35 @@ function GetGSServers()
 		NewItem.Created();
 		NewItem.iMainSvrListIdx = i;
 		NewItem.bFavorite = true;
-		NewItem.bSameVersion = true;
+		NewItem.bSameVersion = !ServerList[i].bWrongVersion;//true;//1.5 change
 		NewItem.szIPAddr = ServerList[i].IP;
-		NewItem.iPing = 1000;//was 1000 in early versions, at some point post 0.6 was changed to 9000?
+		if ( ( ServerList[i].iPing != 0 ) && ( ServerList[i].iPing != -1 ) && ( ServerList[i].iPing != 1000 ) )//1.5 change
+			NewItem.iPing = ServerList[i].iPing;
+		else
+		{
+			ServerList[i].iPing = 1000;//for sorting - null value is 0 which sorted as better than low ping servers
+			NewItem.iPing = 1000;
+		}
 		NewItem.szName = ServerList[i].ServerName;
-		NewItem.szMap = "";
-		NewItem.iMaxPlayers = 0;
-		NewItem.iNumPlayers = 0;
+		NewItem.szMap = ServerList[i].szMap;//"";//1.5 change
+		if ( ServerList[i].iMaxPlayers == 0 )//1.5 change
+		{
+			//for sorting: servers that didn't respond sort the same as servers with 0 players
+			//so set num players in the array as -1 to sort worse
+			ServerList[i].iNumPlayers = -1;
+			NewItem.iMaxPlayers = 0;
+			NewItem.iNumPlayers = 0;
+		}
+		else
+		{
+			NewItem.iMaxPlayers = ServerList[i].iMaxPlayers;
+			NewItem.iNumPlayers = ServerList[i].iNumPlayers;
+		}
 		NewItem.bLocked = ServerList[i].Locked;
-		NewItem.bDedicated = true;
+		NewItem.bDedicated = true;//todo: grab dedicated info from client beacon receiver

```

Comment:
Default assume it's dedicated.  In the future, issue #30 will have us want to differentiate.